src/main/java/io/github/syst3ms/skriptparser/expressions/ExprContextValue.java

```diff
@@ -0,0 +1,73 @@
+package io.github.syst3ms.skriptparser.expressions;
+
+import io.github.syst3ms.skriptparser.Main;
+import io.github.syst3ms.skriptparser.lang.Expression;
+import io.github.syst3ms.skriptparser.lang.TriggerContext;
+import io.github.syst3ms.skriptparser.parsing.ParseContext;
+import io.github.syst3ms.skriptparser.registration.ContextValueManager;
+import org.jetbrains.annotations.Nullable;
+
+import static io.github.syst3ms.skriptparser.registration.ContextValueManager.ContextValue;
+
+/**
+ * A specific context value.
+ * Refer to the documentation of the event to see which values can be used.
+ *
+ * @name Context Value
+ * @pattern [the] (past|future|) context-<.+>
+ * @since ALPHA
+ * @author Mwexim
+ */
+public class ExprContextValue implements Expression<Object> {
+
+	static {
+		Main.getMainRegistration().addExpression(
+			ExprContextValue.class,
+			Object.class,
+			false,
+			3,
+			"[the] (1:past|2:future|3:) context-<.+>"
+		);
+	}
+
+	private String name;
+	private int timeline;
+	private ContextValue<?> value;
+
+	@Override
+	public boolean init(Expression<?>[] vars, int matchedPattern, ParseContext parseContext) {
+		name = parseContext.getMatches().get(0).group();
+		if (parseContext.getParseMark() == 1) timeline = -1;
```

Comment:
I'm aware I didn't specify proper code formatting rules anywhere, but if/else if/else chains without curly brackets are not acceptable. Prefer adding brackets, or perhaps even using a switch statement.

My rules are as such : 

1. Not adding braces is allowed when the code within the block spans a single line (i.e no loops or other control flow statements), and in the case of if statements, there must also not be any else if/else clause.
2. In all other cases, braces are mandatory.